### PR TITLE
Test for reference UTxO with SimpleScripts

### PR DESCRIPTION
**Pull request by mkoura**

### CODE REVIEW

File: cardano_node_tests/tests/test_scripts.py [hunk 1]
```diff
@@ -110,18 +110,60 @@ def multisig_tx(
...
 
+def _create_reference_utxo(
+    temp_template: str,
+    cluster_obj: clusterlib.ClusterLib,
+    payment_addr: clusterlib.AddressRecord,
+    dst_addr: clusterlib.AddressRecord,
+    script_file: Path,
+    amount: int,
+) -> Tuple[clusterlib.UTXOData, clusterlib.TxRawOutput]:
+    """Fund a Plutus script and create the locked UTxO, collateral UTxO and reference script."""
```
Comment (saratomaz) [inline cardano_node_tests/tests/test_scripts.py:133]: update description

Comment (saratomaz) [inline cardano_node_tests/tests/test_scripts.py:133]: update description

**Review by saratomaz — CHANGES_REQUESTED** [review]
update description

File: cardano_node_tests/tests/test_scripts.py (hunk 1 above)
Comment (mkoura) [reply cardano_node_tests/tests/test_scripts.py:133]: What's wrong with the description?

Comment (mkoura) [reply cardano_node_tests/tests/test_scripts.py:133]: I get it, it's not plutus :-)

**Pull request merged**